api/v2/performanceprofile_validation.go

```diff
@@ -253,3 +255,39 @@ func (r *PerformanceProfile) validateNUMA() field.ErrorList {
 
 	return allErrs
 }
+
+func (r *PerformanceProfile) validateNet() field.ErrorList {
+	var allErrs field.ErrorList
+
+	if r.Spec.Net == nil {
+		return allErrs
+	}
+
+	if *r.Spec.Net.UserLevelNetworking && r.Spec.CPU.Reserved == nil {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("spec.net"), r.Spec.Net, "Can not set network devices queues count without specifiying spec.cpu.reserved"))
+	}
+
+	for _, device := range r.Spec.Net.Devices {
```

Comment:
Net can be NULL?

Reply:
there is a check about that returns if Net == NULL